packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.routing.js

```diff
@@ -15,12 +15,9 @@ export default /* @ngInject */ ($stateProvider) => {
         .injector()
         .getAsync('backup')
         .then((backup) => {
-          if (backup.isInactive()) {
+          if (backup.isInactive() || backup.isLegacy()) {
             return { state: 'app.dedicatedClouds.datacenter.backup.new' };
           }
-          if (backup.isLegacy()) {
-            return { state: 'app.dedicatedClouds.datacenter.backup.legacy' };
```

Comment:
What about this state, is it still used ?

Reply:
Its not being used. `Legacy` now is just a banner while creating `new` backup. We handle both flows with some `ng-if`s